formbricks/formbricks
# fix: docker compose upload volume mapping

## What does this PR do?



Docker upload volume is now in `/home/nextjs/apps/web/uploads/`



![image](https://github.com/formbricks/formbricks/assets/20502130/6371dc8b-2be3-421a-803d-aafe6e7d9347)


## Type of change




## How should this be tested?



- Start a docker compose project, upload a file and see where is it (in the container). It's in `/home/nextjs/apps/web/uploads/`

## Checklist



### Required


### Appreciated
